doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +95,25 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
 
 The Cloud Services and the Cloud Resources Brokers provide value-added services in addition to the fundamental capabilities like service and resource discovery.  These Brokers are critical for a multi-cloud environment to function and utilise cloud specific plugins to perform the necessary activities. These Brokers can, for example, provision and manage environments with resources and services for Machine Learning (ML) services, Augmented/Virtual Reality, or specific industries.
 
+<a name="8.2.4"></a>
+### 8.2.4 Aspects of Multi-Cloud Security
+As cloud infrastructures become a key element in the telecommunication operator ecosystem, particularly with the 5G rollout becoming a critical business necessity, cloud-focused threats and associated adversarial behaviours, methods, tools, and strategies that cyber threat actors use to plan and execute are part of the attack surface landscape.
+ 
+In an ecosystem comprised of different network domains, products and business partnerships, the responsibility for managing these different cloud environments necessary to support 5G use cases falls to different organisations, creating a new level of complexities and a new range of security risks. In such an environment, there are additional security principles to be considered.  These principles, see Table 2 below, drawn from the collaboration with GSMA FASG.
+  
+| Multi-cloud Security Principle | Description |
+| ------------------------------ | ----------- |
+| Policy synchronization | Consistency in applying the right security policies across environments, services, interfaces, and configured resources |
+| Visibility | A common data model approach to capture events and behaviours across all the key compute, storage, network, and applications resources, environments, virtualised platforms, containers and interfaces |
```

Comment:
Across different enterprises, a common data model for "capture" may not be feasible at least for a while until standards are formulated and their compliance becomes standard practice.  Can we substitute "share" for "capture"?

Reply:
@pgoyal01, I agree it is aspirational. Good idea to use "share".  I am changing it that way.